packages/tables/resources/views/components/header-cell.blade.php

```diff
@@ -53,7 +53,13 @@ class="fi-ta-header-cell-label text-sm font-semibold text-gray-950 dark:text-whi
         @if ($sortable)
             <x-filament::icon
                 :alias="$activelySorted && $sortDirection === 'asc' ? 'tables::header-cell.sort-asc-button' : 'tables::header-cell.sort-desc-button'"
```

Comment:
I don't want to remove the icon, as then I do not think it would be clear which columns are sortable.

One idea you could PR is a new alias here: `tables::header-cell.sort-button`, and then you can register your own icon for the "not sorted" state: https://filamentphp.com/docs/3.x/support/icons#replacing-the-default-icons

Reply:
This seems to be the best of both worlds! I didn't know of this feature. Awesome! I'll implement this in the PR